draft-ietf-httpbis-priority.md

```diff
@@ -77,17 +77,23 @@ To provide meaningful presentation of a document at the earliest moment, it is
 important for an HTTP server to prioritize the HTTP responses, or the chunks of
 those HTTP responses, that it sends.
 
-HTTP/2 ({{!HTTP2=RFC7540}}) provides such a prioritization scheme. A client sends a
-series of PRIORITY frames to communicate to the server a "priority tree"; this
-represents the client's preferred ordering and weighted distribution of the
-bandwidth among the HTTP responses. However, the design and implementation of
-this scheme has been observed to have shortcomings, explained in {{motivation}}.
-
-This document defines the Priority HTTP header field that can be used by both
-client and server to specify the precedence of HTTP responses in a standardized,
-extensible, protocol-version-independent, end-to-end format. Along with the
-protocol-version-specific frame for reprioritization, this prioritization scheme
-acts as a substitute for the original prioritization scheme of HTTP/2.
+RFC 7540 {{?RFC7540}} stream priority allowed a client to send a series of
+priority signals that communicate to the server a "priority tree"; this
+represents the client's preferred relative ordering and weighted distribution of
+the bandwidth among HTTP responses. Servers could use these priority signals as
+input into prioritization decision making.
+
+The design and implementation of RFC 7540 stream priority was observed to have
+shortcomings, explained in {{motivation}}.
+({{!HTTP2=I-D.ietf-httpbis-http2bis}}) has deprecated the use of these stream
+priority signals.
+
+This document describes an extensible scheme for prioritizing HTTP responses
+that uses absolute values. It defines the Priority HTTP header field that can be
+used by both client and server to specify the precedence of HTTP responses in a
+standardized, extensible, protocol-version-independent, end-to-end format. It
+also defines version-specific frame for reprioritization. This prioritization
```

Comment:
```suggestion
also defines a version-specific frame for reprioritization. This prioritization
```
unless it defines multiple, then it should be "frames", of course

Reply:
Not sure this should be addressed here, but above you say this is protocol-version-independent, while the reprioritization frame is not for H1.

Reply:
This second comment I don't believe is an issue.  "[This document] defines the Priority HTTP header field ... in a ... protocol-version-independent, end-to-end format. [This document] **also** defines version-specific frames for reprioritization."  So, a version-independent header field as well as version-specific frames.

In fact, it sounds like what we should clarify is the antecedent of "it" in the second sentence.

Reply:
Seems like I stupidly missed the "version-specific" part... I agree that resolves my comment.

Reply:
> unless it defines multiple, then it should be "frames", of course

yes I think its more accurate to use the plural form here, since we define H2 and H3 PRIORITY_UPDATE frames

> In fact, it sounds like what we should clarify is the antecedent of "it" in the second sentence.

I'll try and accomodate that too in a commit

Reply:
fixed in ca3c21e59a6d80ca66e1afcb0ccb10c650b53d04